rama-http-types/src/body_ext.rs

```diff
@@ -1,23 +1,57 @@
 use crate::dep::http_body_util::BodyExt;
+use bytes::Bytes;
```

Comment:
```suggestion
use rama_core::bytes::Bytes;
```

Reply:
This doesn't work as rama_core::bytes is not resolved.

```
error[E0432]: unresolved import `rama_core::bytes`
 --> rama-http-types/src/body_ext.rs:2:16
  |
2 | use rama_core::bytes::Bytes;
  |                ^^^^^ could not find `bytes` in `rama_core`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `rama-http-types` (lib) due to 1 previous error
```